Source/JavaScriptCore/runtime/JSONObject.cpp

```diff
@@ -1063,13 +1063,13 @@ void FastStringifier<CharType>::append(JSValue value)
 
         auto charactersCopySameType = [&](auto span, auto* cursor) ALWAYS_INLINE_LAMBDA {
 #if (CPU(ARM64) || CPU(X86_64)) && COMPILER(CLANG)
-            constexpr size_t stride = 16 / sizeof(CharType);
+            constexpr size_t stride = SIMD::stride<CharType>;
```

Comment:
Maybe with this new round of optimization, it will finally be the time to unify the JSC and WTF json parsers :).

Reply:
I'm having a bit concern about it. JSC side JSON is super optimized with JSC object etc., so this is very hard to unify them without regressing JSC performance, and given that WTF side is not so sensitive to performance, I think rather separating them would be simpler.

Reply:
Yeah, that has been the concern all along :). I still believe there is a nice templated chunk that can be shared for the lexing/parsing, but it is certainly a reach.